spec/services/discourse_service_spec.rb

```diff
@@ -152,4 +152,82 @@
       WebMock.disable_net_connect!
     end
   end
+
+  describe ".create_email_token" do
+    it "creates a email token" do
+      user = FactoryBot.create(:user)
+
+      discourse = DiscourseService.new(user)
+
+      response = {
```

Comment:
You are making a literal object and then asserting over its properties, how this test is valuable?

Reply:
We are making a request inside the method, and we are stubbing the request with this result (the same result that comes from the actual API).